hphp/runtime/base/code-coverage.cpp

```diff
@@ -129,6 +160,42 @@ Array CodeCoverage::Report(bool sys /* = true */) {
   return ret;
 }
 
+Array CodeCoverage::ReportExecutable() {
```

Comment:
Both this and ReportDeadCode could go since all the bookkeeping would happen in m_hits, and be reported by Report.